src/core/data-tests.js

```diff
@@ -48,10 +47,9 @@ function toListItem(href) {
     .split("-")
     .find(part => part === "manual");
   if (isManualTest) {
-    const manualPerformEmoji = hyperHTML.bind(
-      document.createElement("span")
-    )` 💪 `;
-    manualPerformEmoji.setAttribute(
+    const manualPerformEmoji = document.createElement("span")
+      manualPerformEmoji.innerHTML = `💪`;
```

Comment:
Nit: note that the indentation will get reset by [prettier.js](https://github.com/prettier/prettier)

Reply:
oh completely forgot about that. Apart from that any more changes you would like to see ?

Reply:
Lastly, we would need to add tests to tests/spec/core/data-tests-spec.js, that check all the new changes we've made. 

So, pretty close to being done here! Thanks for hanging in there!